libraries/monitor/README.md

```diff
@@ -339,3 +339,16 @@ This function will:
  * shut down and flush monitoring
  * call `process.exit` on success or failure
 Note, then, that the function **does not return**.
+
+### Testing
+
+If the `mock` option is true, then all messges logged are available from the `messages` property on the MonitorManager, in the format `{Type, Fields, Logger}`.
+These messages can be cleared with the `reset()` method -- generally done in a Mocha `setup` function.
+This allows tests to assert that a message was logged correctly:
```

Comment:
:+1: good idea. I wasn't able to come up with a nice way to do this.

Reply:
Well, you did really -- it just wasn't documented and I implemented it in a worse way, then discovered your implementation and documented it :)